draft-ietf-tls-tls13.md

```diff
@@ -2612,6 +2625,11 @@ facilitates implementation as described below). Servers MUST check
 that it is the last extension and otherwise fail the handshake with an
 "illegal_parameter" alert.
 
+When the syntax of this extension is correct but the server cannot
+verify at least one binder because of some configuration or state
+changes, the server SHOULD fall back to a full 1-RTT handshake if a
+"key_share" extension is supplied.
```

Comment:
I agree with MT. The order of operations is:

- Determine whether you are going to PSK and select the PSK
- If you are resuming, check the binder.

If the binder checks fails, fail the handshake